api/model/src/main/java/org/projectnessie/api/v2/TreeApi.java

```diff
@@ -324,11 +325,18 @@ CommitResponse commitMultipleOperations(
    * table-metadata) plus the per-Nessie-reference/hash specific part (Iceberg: snapshot-ID,
    * schema-ID, partition-spec-ID, default-sort-order-ID).
    *
+   * <p>Throws a {@code AccessCheckException} if access checks fail. Note that if the content object
+   * does not exist <em>and</em> the access checks fail, an {@code AccessCheckException} will be
+   * thrown, not a {@link NessieContentNotFoundException}.
+   *
    * @param key the {@link ContentKey}s to retrieve
    * @param ref named-reference to retrieve the content for
    * @param withDocumentation whether to return the documentation, if it exists.
+   * @param forWrite If set to 'true', access control checks will check for write/create privilege
+   *     in addition to read access checks.
    * @return list of {@link GetMultipleContentsResponse.ContentWithKey}s
-   * @throws NessieNotFoundException if {@code ref} or {@code hashOnRef} does not exist
+   * @throws NessieNotFoundException if the content object or if {@code ref} or {@code hashOnRef}
```

Comment:
The new wording is weird; and besides, `hashOnRef` is not a parameter of this method.